portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,32 +32,29 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {    
-    //reading GET request max-comments parameter to determine how much comments include in the response
     String commentsAction = getParameter(request, "comments", "current");
     String newestStr = getParameter(request, "newest", "");
     String oldestStr = getParameter(request, "oldest", "");
     String sizeStr = getParameter(request, "size", "");
     String jsonArray = "";
-    System.out.println(newestStr);
-    System.out.println(oldestStr);
     long oldest = Long.MAX_VALUE;
     long newest = 0;
     int size = 5;
     
     try{
       oldest = Long.parseLong(oldestStr);
     }catch(NumberFormatException e){
-      System.out.println("Cant parse oldest");
+      e.printStackTrace();
     }
     try{
       newest = Long.parseLong(newestStr);
     }catch(NumberFormatException e){
-      System.out.println("Cant parse newest");
+      e.printStackTrace();
     }
     try{
       size = Integer.parseInt(sizeStr);
     }catch(NumberFormatException e){
-      System.out.println("Cant parse size");
+      e.printStackTrace();
     }
```

Comment:
I'd prefer to merge them in a single try-catch block. Understanding exactly which integer fails to parse shouldn't be a problem if we get the stacktrace (which should include the line number).